globals.d.ts

```diff
@@ -0,0 +1,5 @@
+declare var MIDI: any;
```

Comment:
Make sure to file issue for resolving this (please link to it in your reply to this comment)

Reply:
Done. Issue #6351